src/screens/benefit/BenefitFormScreenWrapper.tsx

```
@@ -0,0 +1,53 @@
+import { getUser } from "../../services/auth/auth";
+import { getOne } from "../../services/benefit/benefits";
+
+const BenefitFormScreenWrapper: React.FC = () => {
+  const { id, bpp_id } = useParams<{ id: string; bpp_id: string }>();
+  const location = useLocation();
+  const navigate = useNavigate();
+  const state = location.state || {};
+
+  const [data, setData] = useState(state);
+
+  useEffect(() => {
```

ESLint

[error] Replace `··` with `↹`

(prettier/prettier)